src/sentry/monitors/tasks/detect_broken_monitor_envs.py

```diff
@@ -82,6 +85,24 @@ def update_user_monitor_dictionary(
         user_monitor_entry["environment_names"].append(environment_name)
 
 
+def get_user_emails_from_monitor(
+    monitor: Monitor,
+):
+    if monitor.owner_user_id:
+        organization_member = OrganizationMember.objects.get(
+            user_id=monitor.owner_user_id, organization_id=monitor.organization_id
+        )
+        user_emails = [organization_member.user_email]
+    elif monitor.owner_team_id:
+        team = Team.objects.get_from_cache(id=monitor.owner_team_id)
+        user_emails = [user.user_email for user in team.member_set]
```

Comment:
Wondering if we should be catching `<model>.DoesNotExist` here and falling back to the whole project

Reply:
ooh yea lets do that